VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -160,38 +200,75 @@ final public class AuthenticationController
     
     // MARK: - Private Authentication
     
+    /**
+     **(PRIVATE: Vimeo Use Only)**
+     Log in with an email and password
+     
+     - parameter email:      a user's email
+     - parameter password:   a user's password
+     - parameter completion: handler for authentication success or failure
+     */
```

Comment:
Can we use the `@available` annotation similar to how we add deprecation warnings?